test/infrastructure/RepositoryFactory.spec.ts

```diff
@@ -63,10 +63,10 @@ describe('RepositoryFactory', () => {
             createBlockRepository(): BlockRepository {
                 return instance(repositoryMock);
             }
-        })("http://localhost:3000");
+        })('http://localhost:3000');
 
         expect(counter).to.be.equals(0);
-        repositoryFactory.getGenerationHash().subscribe(gh => {
+        repositoryFactory.getGenerationHash().subscribe((gh) => {
             expect(counter).to.be.equals(1);
             expect(gh).to.be.equals('aaaa');
             repositoryFactory.getGenerationHash().subscribe(gh => {
```

Comment:
Thanks, @rg911 for the cleanup. Question, why the braces in arrow function's parameter when it is not necessary?